app/models/assignment.rb

```diff
@@ -128,7 +132,7 @@ def as_platform_json
         'organization' => Organization.current.name,
         'sid' => submission_id,
         'created_at' => updated_at,
-        'content' => solution,
+        'content' => submittable_solution,
```

Comment:
we should send both user and submittable solution, but this will be introduced when the refactor is done in classroom